packages/web-components/fast-components/src/flipper/flipper.styles.ts

```diff
@@ -48,8 +48,8 @@ export const FlipperStyles = css`
         transition: all 0.1s ease-in-out;
     }
 
-    .next,
-    .previous {
+    [direction="next"],
+    [direction="previous"] {
```

Comment:
@vertonghenb These classes are fine here (`next`, `previous`). They are within the shadow DOM  rather than on the host so they should be fine. Adding these back in will solve the issue.

Reply:
makes sense, the same as the ones for `tabs` thanks!

Reply:
Yeah, and thanks for the note on vertical/horizontal in tabs. I think we can follow up on that one if you're seeing issues there as well.